RunGoogleClassroom.py

```diff
@@ -36,21 +36,15 @@ def Parse_Args(filter1):
     parser.set_defaults(output=True)
     parser.set_defaults(file=True)
     args = parser.parse_args()
-    parsed = True
```

Comment:
The arg parser will complete only if it is successful, so I didn't think we needed the extra status.

Reply:
Thanks for the cleanup, you can tell Python is not my primary language.  It's my "hacking stuff together" language.  I bet you can clean up the sorting by name logic I just added. :)